recipes-kernel/kernel-tests/kernel-performance-tests-files/run-cyclictest

```diff
@@ -17,61 +17,67 @@ function add_system_info()
 	# save info on the current security mitigations
 	echo "# Security vulnerabilities settings: "
 	if [ -d "/sys/devices/system/cpu/vulnerabilities" ]; then
-	    for f in  /sys/devices/system/cpu/vulnerabilities/*; do
+		for f in  /sys/devices/system/cpu/vulnerabilities/*; do
 		VULN=$(basename "$f")
 		if [ -f "$f" ]; then
-		    echo -n -e "#     $VULN:\t"
-		    cat "$f"
+			echo -n -e "#	 $VULN:\t"
+			cat "$f"
 		fi
-	    done
+		done
 	else
-	    echo "#     vulnerability information not published"
+		echo "#	 vulnerability information not published"
 	fi
 
 	# save info on the current C state settings
 	echo "# C-states settings: "
 	for CPU in /sys/devices/system/cpu/cpu[0-9]; do
-	    NCPU=$(basename "$CPU")
-	    echo "#     [$NCPU]"
-	    for CSTATE in $CPU/cpuidle/state[0-9]; do
+		NCPU=$(basename "$CPU")
+		echo "#	 [$NCPU]"
+		for CSTATE in $CPU/cpuidle/state[0-9]; do
 		NSTATE=$(basename "$CSTATE")
 		if [ -f "$CSTATE/name" ] && [ -f "$CSTATE/disable" ]; then
-                    NAME=$(cat "$CSTATE/name" 2>/dev/null)
-                    STATUS=$(cat "$CSTATE/disable" 2>/dev/null)
+					NAME=$(cat "$CSTATE/name" 2>/dev/null)
+					STATUS=$(cat "$CSTATE/disable" 2>/dev/null)
 
-                    echo -n "#     $NSTATE: $NAME: "
-                    if [ "$STATUS" = "0" ]; then
-                        echo "enabled"
-                    else
-                        echo "disabled"
-                    fi
+					echo -n "#	 $NSTATE: $NAME: "
+					if [ "$STATUS" = "0" ]; then
+						echo "enabled"
+					else
+						echo "disabled"
+					fi
 		else
-                    echo "#     information not available"
+					echo "#	 information not available"
 		fi
-	    done
+		done
 	done
-    } >> "$1"
+	} >> "$1"
 }
 
 function run_cyclictest()
 {
-    LOG="$LOG_DIR/cyclictest-$1-`date +'%Y_%m_%d-%H_%M_%S'`.log"
-    cyclictest --smp --priority=98 --mlockall --interval=997 --quiet --duration="$TEST_DURATION" --histofall=1000 --histfile="$LOG" > /dev/null
-    add_system_info "$LOG"
-    python3 upload_cyclictest_results.py "-i $LOG" "-s dso-db-influx-1.ni.systems" "-p 8086"
+	LOG="$LOG_DIR/cyclictest-$1-`date +'%Y_%m_%d-%H_%M_%S'`.log"
+	INFLUXDB_INFO="/home/admin/.influxdb.info"
+	cyclictest --smp --priority=98 --mlockall --interval=997 --quiet --duration="$TEST_DURATION" --histofall=1000 --histfile="$LOG" > /dev/null
+	add_system_info "$LOG"
+	if [[ -f "$INFLUXDB_INFO" ]]; then
+		source "$INFLUXDB_INFO"
```

Comment:
Adding a comment to block submission for now. The changes to our test mobilize plans need to go in first.